docs/pages/messages/package-task-in-single-package-mode.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Package Task In Single Package Mode
+description: "package-task-in-single-package-mode error page"
+---
+
+# Package Task In Single Package Mode
+
+In single package mode, there cannot be multiple packages in your repository. Therefore, declaring a task in the
+`turbo.json` with a specified package name is not permitted.
+
+## Example
+
+```json
+{
+  "pipeline": {
+    "app#build": {
+      "cache": true
+    }
+  }
+}
+```
+
+## Solution
+
+Remove the package name from the task declaration.
+
+```json
+{
+  "pipeline": {
+    "build": {
+      "cache": true
+    }
+  }
+}
+```
+
+Alternatively, you can disable single package mode by [specifying the workspaces in the repository](/repo/docs/getting-started/existing-monorepo#configure-workspaces).
```

Comment:
This line seems droppable. I *can* do this but it's probably not what I meant to do if I run into this error while I'm working in a single-package workspace.

Reply:
In my experience, turbo can sometimes accidentally infer a single package repository, and this will be the first error that you will encounter in that situation. So it might be good to mention this

Reply:
Ah, okay, that makes sense. In that case, would it make more sense to make it more in the positive lane. Like: "Alternatively, you can [configure workspaces for your repository] (link to doc)." Thinking I probably would've caught on better if it were that direction.